### DISCUSSION TITLE
System.Text.Json update to 8.0.5

### DISCUSSION THREAD

**Discussion by julhenau** [reactiveui/ReactiveUI#3927] [General]
Hi everyone,
Do you have any ETA for this to be included in the next update of ReactiveUI.WPF ?
Thanks

**Comment by frederic-gerlach-bitifeye-com** [comment]
According to https://www.nuget.org/packages/system.text.json/#versions-body-tab System.Text.Json 8.0.4 has at least one vulnerability with high severity that is fixed in 8.0.5.

It looks like renovate does not see that issue so a manual fix/update would be required.

**Reply by glennawatson** [reply]
Renovate just waits a week for new packages. Nothing stopping you from putting a upgrade in yourself to stop the vulnerability warning.

**Reply by frederic-gerlach-bitifeye-com** [reply]
Thanks for the explanation. I did not want to push a manual change before understanding why the automated system for that already in place does not do it.

I still do not 100 % understand how renovate running weekly would explain that, because System.Text.Json version 8.0.5 was released 9 days ago. Could there be some configuration issue? Or am I just interpreting "a week" too literally?

**Reply by dpvreony** [reply]
there's a 10 day stability requirement on new packages. might be something we want to look at for MS packages

**Comment by julhenau** [comment]
Hi,
Is there any update about this?

**Reply by glennawatson** [reply]
Just explicitly upgrade the system text json version in your project by adding a PackageReference. You aren't bound by what rxui has that's just a minimum requirement.
